tests/rustdoc/enum/enum-variant-non_exhaustive.rs

```diff
@@ -0,0 +1,17 @@
+// regression test for https://github.com/rust-lang/rust/issues/142599
+
+#![crate_name = "foo"]
+
+//@ snapshot type-code 'foo/enum.Type.html' '//pre[@class="rust item-decl"]/code'
+pub enum Type {
+    #[non_exhaustive]
+    // attribute that should not be shown
+    #[warn(unsafe_code)]
+    Variant,
+}
+
+// we would love to use the `following-sibling::` axis
```

Comment:
Does this comment still make sense? Current code is fine and I'm not sure to understand what `follow-sibling` added value would be here.

Reply:
it would allow us to select "the code block in the aliased type section" instead of "the second code block" which would be robust against us adding additional code blocks or rearranging them.

Reply:
Fair enough!